task/12-katas-3-tasks.js

```diff
@@ -25,18 +25,59 @@
  *   'CLASS'     => true
  *   'ARRAY'     => true   (first column)
  *   'FUNCTION'  => false
- *   'NULL'      => false 
+ *   'NULL'      => false
  */
 function findStringInSnakingPuzzle(puzzle, searchStr) {
-    throw new Error('Not implemented');
+    function nfunc(current, step) {
+        let save = data[current.y][current.x];
+        data[current.y][current.x] = "";
+
+        if (step == search.length)
```

Comment:
"operator === is always preferable
sometimes ''==' may cause an error
(check at console :
'0' == 0
and '0' === 0)"